neopixel.py

```diff
@@ -101,6 +101,9 @@ def __init__(self, num_leds, state_machine, pin, mode="RGB", delay=0.0003):
         self.num_leds = num_leds
         self.delay = delay
         self.brightnessvalue = 255
+        self.dma = rp2.DMA()
+        DATA_REQUEST_INDEX = (pin << 3) + state_machine
+        self.dma_ctrl = self.dma.pack_ctrl(size=2, inc_write=False, treq_sel=DATA_REQUEST_INDEX)
```

Comment:
It would be also nice to add some comment here for later development. Right now, it's not entirely clear from the code alone where these indexes come from.